spec/index.html

```diff
@@ -653,19 +670,27 @@ <h3>Algorithm</h3>
 
   <section id="issue-identifier" class="algorithm">
     <h2>Issue Identifier Algorithm</h2>
-    <!--section id="issue-identifier-overview" class="informative">
+
+    <p>This algorithm issues a new <a>blank node identifier</a> for
+      a given existing <a>blank node identifier</a>. It also updates
+      state information that tracks the order in which new
+      <a>blank node identifiers</a> were issued.</p>
+
+    <section id="issue-identifier-overview">
       <h3>Overview</h3>
-    </section-->
+
+      <p>The algorithm maintains an <a>issued identifiers map</a> to
+        relate an existing <a>blank node identifier</a> from the <a>input dataset</a>
+        to a new <a>blank node identifier</a> using a given <a>identifier prefix</a>
+        (`_:c14n`) with new identifiers issued by appending an incrementing number.
+        For example, when called for a blank node identifier such as `_:a`,
```

Comment:
That would be more consistent with our other examples, but I realized that `_:b` is used for a temporary issuer in the main algorithm. This could be seen as confusing when used in examples later on – alternatively, it also shows that  there is no danger of overlap from existing identifiers to canonical identifiers.